src/actions/component.ts

```diff
@@ -88,7 +90,18 @@ export const createComponent = async (
     });
   }
 
-  loader.stop();
-  loader.clear();
-  console.log(blue(`Creation of ${formatedComponentName} component completed`));
+  if (options['withHooks']) {
+    loader.clear();
+    loader.text = 'Creating hook';
+    await createHooks({
+      isOnlyJs,
+      componentPath,
+    });
+
+    loader.stop();
+    loader.clear();
+    console.log(
+      blue(`Creation of ${formatedComponentName} component completed`),
+    );
```

Comment:
esto no deberia ir dentro del if que pasa si no viene la opcion de with hooks ? 
```suggestion
    loader.stop();
    loader.clear();
    console.log(
      blue(`Creation of ${formatedComponentName} component completed`),
    );
```